web/can_data/CAN2.ini

```diff
@@ -0,0 +1,165 @@
+[View_Vehicles]
```

Comment:
please add `.ini` files to the `.gitignore` file in the top level of the directory. These files are generated by Vector DB++ and are useless to keep in git.